tasks.py

```diff
@@ -282,8 +282,8 @@ def content_excludes():
     return output
 
 
-@task(help={'filename': "Output filename (default = 'data.json')"})
-def export_records(c, filename='data.json'):
+@task(help={'filename': "Output filename (default = 'data.json')", 'overwrite': "Overwrite existing files without asking first (default False)"})
+def export_records(c, filename='data.json', overwrite=False):
```

Comment:
Request: Please add type 'bool'

Reply:
We are currently typing out the whole code base. There is no enforcement but that will be introduced in a bit.

Reply:
If you mean that I should change the function definition to the following:
```
def export_records(c, filename: str = 'data.json', overwrite: bool = False):
```
then that won't work with the current tests `invoke`: it fails with the error `ValueError: Function has keyword-only parameters or annotations, use inspect.signature() API which can support them`:

```
Traceback (most recent call last):
  File "/home/ubuntu/.local/bin/invoke", line 8, in <module>
    sys.exit(program.run())
  File "/home/ubuntu/.local/lib/python3.10/site-packages/invoke/program.py", line 373, in run
    self.parse_collection()
  File "/home/ubuntu/.local/lib/python3.10/site-packages/invoke/program.py", line 465, in parse_collection
    self.load_collection()
  File "/home/ubuntu/.local/lib/python3.10/site-packages/invoke/program.py", line 696, in load_collection
    module, parent = loader.load(coll_name)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/invoke/loader.py", line 76, in load
    module = imp.load_module(name, fd, path, desc)
  File "/usr/lib/python3.10/imp.py", line 235, in load_module
    return load_source(name, filename, file)
  File "/usr/lib/python3.10/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 719, in _load
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/InvenTree/tasks.py", line 291, in <module>
    def export_records(c, filename: str = 'data.json', overwrite: bool = False):
  File "/home/ubuntu/.local/lib/python3.10/site-packages/invoke/tasks.py", line 357, in inner
    obj = klass(
  File "/home/ubuntu/.local/lib/python3.10/site-packages/invoke/tasks.py", line 76, in __init__
    self.positional = self.fill_implicit_positionals(positional)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/invoke/tasks.py", line 167, in fill_implicit_positionals
    args, spec_dict = self.argspec(self.body)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/invoke/tasks.py", line 153, in argspec
    spec = inspect.getargspec(func)
  File "/usr/lib/python3.10/inspect.py", line 1245, in getargspec
    raise ValueError("Function has keyword-only parameters or annotations"
ValueError: Function has keyword-only parameters or annotations, use inspect.signature() API which can support them
``

Putting the hint in the help text doesn't really make much sense, since it does not apply to the CLI (where the flag is either there, or not..)